specification/sql/resource-manager/readme.md

```diff
@@ -31,6 +31,135 @@ tag: package-composite-v4
 
 The following packages may be composed from multiple api-versions.
 
+### Tag: package-composite-v5
+
+These settings apply only when `--tag=package-composite-v5` is specified on the command line.
+
+This section contains the "composite-v5" set of APIs, which is composed from a selection of api-versions that will remain backwards compatible with "v5" clients such as .NET SDK Microsoft.Azure.Management.Sql version 1.44.3.0-preview.
+
+APIs must only be added to this section when the API is publicly available in at least 1 production region and at least 1 generated client has been tested end-to-end.
+
+
+``` yaml $(tag) == 'package-composite-v5'
+input-file:
+- Microsoft.Sql/stable/2014-04-01/backups_legacy.json
+- Microsoft.Sql/stable/2014-04-01/dataMasking.json
+- Microsoft.Sql/stable/2014-04-01/geoBackupPolicies.json
+- Microsoft.Sql/stable/2014-04-01/metrics.json
+- Microsoft.Sql/stable/2014-04-01/recommendedElasticPoolsDecoupled.json
+- Microsoft.Sql/stable/2014-04-01/replicationLinks_legacy.json
+- Microsoft.Sql/stable/2014-04-01/serverCommunicationLinks.json
+- Microsoft.Sql/stable/2014-04-01/serviceObjectives.json
+- Microsoft.Sql/stable/2014-04-01/sql.core_legacy.json
+- Microsoft.Sql/stable/2014-04-01/usages_legacy.json
```

Comment:
after careful verification via operation ids, we found these legacy API endpoints from these 2014 spec files are not covered by the latest version spec file: 2020-08-01-preview.
Thus, as the walkaround, we need to keep these files in separated files (xyz_legacy.json) here to prevent the incompatibility issues from previous versions of SDK. 
We are contacting the owners in SQL org to re-implement these legacy API endpoints in the latest version.

Reply:
still not quite understand why need new legacy file. it is same file as origin one?

Reply:
@akning-ms 

In v5 SDK, we would like to use 2020-08-01 specs files to cover as many API endpoints as possible. However, for some legacy API endpoints in the old stable 2014-04-01 version, we still cannot use the 2020-08-01 specs file to cover. 

The replacing strategy is a little complex. There are four cases:
1. if all the API endpoints in one 2014-04-01 spec file are deprecated, we just remove it from v5 SDK. i.e. we remove - Microsoft.Sql/stable/2014-04-01/connectionPolicies.json
2. if all the API endpoints in one 2014-04-01 spec file are cover by 2020-08-01 spec file(s), we also remove it from v5 SDK. i.e. we remove - Microsoft.Sql/stable/2014-04-01/databaseSecurityAlertPolicies.json
3. if all the API endpoints in one 2014-04-01 spec file are **NOT** cover by **ANY** 2020-08-01 spec file(s), we will keep it in v5 SDK. i.e. we keep - Microsoft.Sql/stable/2014-04-01/dataMasking.json
4. if some of the API endpoints in one 2014-04-01 spec file are **NOT** cover by 2020-08-01 spec file(s) and some are covered, we will create a file called XYZ_legacy to contain all the **un-covered** API endpoints. i.e. we remove - Microsoft.Sql/stable/2014-04-01/backups.json and add - Microsoft.Sql/stable/2014-04-01/backups_legacy.json in v5 SDK.

For case 3 and case 4, we are contacting the owners in SQL org to re-implement these legacy API endpoints in the latest version. In the upcoming v6 SDK, we plan to deprecate all the 2014-04-01 spec files.

We can not auto-generate the 2014-04-01 spec files. Manual modifications are easy to break this legacy stable API and impact on customer stable usage. Thus, we tried our best to NOT modify any change on the files in https://github.com/Azure/azure-rest-api-specs/tree/master/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01

cc @jamestao